kernel/build.gradle.kts

```diff
@@ -2,24 +2,53 @@
  * Build the "kernel" module.
  */
 
-val libDir = "${rootProject.projectDir}/lib"
+plugins {
+    alias(libs.plugins.xtc)
+}
+
+dependencies {
+    xdkDistribution(libs.xdk)
+    xtcModule(project(":auth"))
+    xtcModule(project(":common"))
+    xtcModule(project(":platformDB"))
+}
 
-tasks.register("build") {
-    group       = "Build"
-    description = "Build this module"
+// Process cfg.json template with port overrides if set
```

Comment:
I don't see what this has to do with a build. The configuration file is **run-time only** concern. Why would the build need to know the semantics of the config file?